Introduce `pluginsWithPermission` information (#2213)
backstage/community-plugins: workspaces/rbac/plugins/rbac-backend/README.md

For more information on the available API endpoints accessible to the policy administrators, refer to the [API documentation](./docs/apis.md).

### Configure plugins with permission

In order for the RBAC UI to display available permissions provided by installed plugins, add the corresponding
plugin IDs to the `app-config.yaml`.

You can specify the plugins with permission in your application configuration as follows:

```YAML
permission:
  enabled: true
  rbac:
    pluginsWithPermission:
      - catalog
      - scaffolder
      - permission
    admin:
      users:
        - name: user:default/alice
        - name: group:default/admins
```

> For more information on the available permissions within Showcase and RHDH, refer to the [permissions documentation](./docs/permissions.md).

Reviewer: Hi @dzemanov,

thank you for this additional doc section. But this is a general Backstage repository and we should try to avoid product-related naming here when possible.

I think we can replace Showcase and RHDH here with Backstage. Or you can remove it completely, like:

```suggestion
For more information on the available permissions, refer to the [RBAC permissions documentation](./docs/permissions.md).
```

I recommend also to add also "RBAC" in front of the permission documentation link to make it clear that it's **not** the upstream Backstage permission doc (https://backstage.io/docs/permissions/overview/).

Can you update in this PR also the headlines of the linked [docs/permissions.md](https://github.com/backstage/community-plugins/blob/main/workspaces/rbac/plugins/rbac-backend/docs/permissions.md)?

Thank you!

Author: Good catch, thank you for the review! Updated.
